test/src/main/java/org/apache/accumulo/test/InstanceOperationsIT.java

```diff
@@ -0,0 +1,107 @@
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.accumulo.core.client.Accumulo;
+import org.apache.accumulo.core.client.AccumuloClient;
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.admin.InstanceOperations;
+import org.apache.accumulo.core.client.admin.servers.ServerId;
+import org.apache.accumulo.core.conf.Property;
+import org.apache.accumulo.harness.AccumuloClusterHarness;
+import org.apache.accumulo.miniclusterImpl.MiniAccumuloConfigImpl;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.jupiter.api.Test;
+
+public class InstanceOperationsIT extends AccumuloClusterHarness {
+
+  @Override
+  public void configureMiniCluster(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite) {
+    cfg.setProperty(Property.INSTANCE_ZK_TIMEOUT, "10s");
+    cfg.getClusterServerConfiguration().setNumDefaultCompactors(3);
+    cfg.getClusterServerConfiguration().setNumDefaultScanServers(2);
+    cfg.getClusterServerConfiguration().setNumDefaultTabletServers(1);
+  }
+
+  @SuppressWarnings("deprecation")
+  @Test
+  public void testGetServers() throws AccumuloException, AccumuloSecurityException {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      InstanceOperations iops = client.instanceOperations();
+
+      assertEquals(3, iops.getServers(ServerId.Type.COMPACTOR).size());
+      assertEquals(3, iops.getCompactors().size());
+      assertTrue(validateAddresses(iops.getCompactors(), iops.getServers(ServerId.Type.COMPACTOR)));
+
+      assertEquals(2, iops.getServers(ServerId.Type.SCAN_SERVER).size());
+      assertEquals(2, iops.getScanServers().size());
+      assertTrue(
+          validateAddresses(iops.getScanServers(), iops.getServers(ServerId.Type.SCAN_SERVER)));
+
+      assertEquals(1, iops.getServers(ServerId.Type.TABLET_SERVER).size());
+      assertEquals(1, iops.getTabletServers().size());
+      assertTrue(
+          validateAddresses(iops.getTabletServers(), iops.getServers(ServerId.Type.TABLET_SERVER)));
+
+      assertEquals(1, iops.getServers(ServerId.Type.MANAGER).size());
+      assertEquals(1, iops.getManagerLocations().size());
+      assertTrue(
+          validateAddresses(iops.getManagerLocations(), iops.getServers(ServerId.Type.MANAGER)));
```

Comment:
Could make validateAddresses()  return void.


```suggestion
          validateAddresses(iops.getManagerLocations(), iops.getServers(ServerId.Type.MANAGER));
```

Reply:
Implemented in be26e58